components/suggest/src/schema.rs

```diff
@@ -775,12 +772,6 @@ CREATE TABLE yelp_modifiers(
     PRIMARY KEY (type, keyword)
 ) WITHOUT ROWID;
 
-CREATE TABLE yelp_location_signs(
-    keyword TEXT PRIMARY KEY,
-    need_location INTEGER NOT NULL,
-    record_id TEXT NOT NULL
-) WITHOUT ROWID;
-
```

Comment:
Sorry, I didn't notice this before. This schema here is the v16 schema (see the `const` name above). So this needs to stay, and then these tests will make sure that the proper migrations are applied and the fully migrated schema matches the current schema at the top of the file. That should fix the other test failure.

Reply:
Oh, I see, I revert it.